samples/generic-base/logging/user.env

```diff
@@ -108,6 +108,21 @@
 # NODE_NAME=mynode.example.com
 
 
+# Use port-forwarding during deployment
+# ------------------------------
+# By default, the deployment process will use
+# the public URLs when accessing the APIs for
+# Kibana and Elasticsearch.  However, in some
+# environments, access to the applications 
+# via these URLs is NOT possible from the
+# machine on which the deployment process will
+# be run.  In those situations, set the following
+# property to "true" to force the deployment
+# scripts to use Kubernetes port-forwarding
+# when accessing the Kibana and/or Elasticsearch
+# APIs.
+# LOG_ALWAYS_PORT_FORWARD=false
+
```

Comment:
Same comment about maybe being more succinct and just including a link instead of a paragraph for a single setting.

Reply:
see above